src/arr/compiler/anf-loop-compiler.arr

```diff
@@ -899,12 +930,14 @@ end
 
 fun is-id-occurs(target :: A.Name, e :: J.JExpr) block:
   doc: "Returns true iff `target` occurs in `e`"
+  dummy-js-expr = j-num(0)
   var found = false
   e.visit(J.default-map-visitor.{
```

Comment:
Ditto.  If you're throwing away the result of the visit, then this can change to be the default-iter-visitor also.